README.md

```diff
@@ -1,103 +1,13 @@
-Doctrine Coding Standard
-========================
+# Doctrine Coding Standard
 
 [![Build Status](https://img.shields.io/travis/doctrine/coding-standard/master.svg?style=flat-square)](http://travis-ci.org/doctrine/coding-standard)
 [![Total Downloads](https://img.shields.io/packagist/dt/doctrine/coding-standard.svg?style=flat-square)](https://packagist.org/packages/doctrine/coding-standard)
 [![Latest Stable Version](https://img.shields.io/packagist/v/doctrine/coding-standard.svg?style=flat-square)](https://packagist.org/packages/doctrine/coding-standard)
 
+The Doctrine Coding Standard is a set of [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) rules applied to all Doctrine projects.
```

Comment:
Can we also give attribution here to the Slevomat guys / @kukulich for their amazing work on Slevomat CS? Something like: 
```markdown
Doctrine Coding Standard is heavily based on [Slevomat Coding Standard](https://github.com/slevomat/coding-standard).
```